### PR TITLE
update vbs backup resource

### PR DESCRIPTION
**Pull request by smilemusic**
make testacc TEST=./huaweicloud/ TESTARGS='-run TestAccVBSBackupV2_basic'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/ -v -run TestAccVBSBackupV2_basic -timeout 360m
=== RUN   TestAccVBSBackupV2_basic
--- PASS: TestAccVBSBackupV2_basic (241.04s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud       241.087s

### CODE REVIEW

File: huaweicloud/resource_huaweicloud_vbs_backup_v2_test.go [hunk 1]
```diff
@@ -111,33 +109,54 @@ func testAccCheckVBSBackupV2Exists(n string, configs *backups.Backup) resource.T
...
-  name = "volume_123"
-  description = "first test volume"
-  size = 40
-  cascade = true
+resource "huaweicloud_evs_volume" "volume" {
+  name              = "volume"
+  description       = "my volume"
+  volume_type       = "SATA"
+  size              = 20
+  availability_zone = "cn-north-4a"
```
Comment (ShiChangkuo) [inline huaweicloud/resource_huaweicloud_vbs_backup_v2_test.go]: replace the specified  availability_zone with `OS_AVAILABILITY_ZONE`

File: huaweicloud/resource_huaweicloud_vbs_backup_v2_test.go [hunk 2]
```diff
@@ -111,33 +109,54 @@ func testAccCheckVBSBackupV2Exists(n string, configs *backups.Backup) resource.T
 }
 
 const testAccVBSBackupV2_basic = `
-resource "huaweicloud_blockstorage_volume_v2" "volume_1" {
-  name = "volume_123"
-  description = "first test volume"
-  size = 40
-  cascade = true
+resource "huaweicloud_evs_volume" "volume" {
+  name              = "volume"
```
Comment (ShiChangkuo) [inline huaweicloud/resource_huaweicloud_vbs_backup_v2_test.go]: it's appreciated to use the acctest.RandString to the name of evs and others.

File: website/docs/r/vbs_backup.html.markdown [hunk 3]
```diff
@@ -15,18 +15,35 @@ This is an alternative to `huaweicloud_vbs_backup_v2`
...
+  name              = "volume"
+  description       = "my volume"
+  volume_type       = "SATA"
+  size              = 20
+  availability_zone = "cn-north-4a"
+
+  tags = {
+    foo = "bar"
+    key = "value"
+  }
```
Comment (ShiChangkuo) [inline website/docs/r/vbs_backup.html.markdown]: tags maybe meaningless in this scene.

File: huaweicloud/resource_huaweicloud_vbs_backup_v2.go [hunk 4]
```diff
@@ -167,13 +167,12 @@ func resourceVBSBackupV2Read(d *schema.ResourceData, meta interface{}) error {
...
 	d.Set("name", n.Name)
-	d.Set("description", n.Description)
 	d.Set("status", n.Status)
 	d.Set("availability_zone", n.AvailabilityZone)
 	d.Set("snapshot_id", n.SnapshotId)
-	d.Set("service_metadata", n.ServiceMetadata)
 	d.Set("size", n.Size)
-	d.Set("container", n.Container)
+	d.Set("object_count", n.ObjectCount)
+	d.Set("created_at", n.CreatedAt.String())
```
Comment (ShiChangkuo) [inline huaweicloud/resource_huaweicloud_vbs_backup_v2.go]: suggest to use `n.CreatedAt.Format(time.RFC3339)`

File: website/docs/r/vbs_backup.html.markdown (hunk 3 above)
Comment (smilemusic) [reply website/docs/r/vbs_backup.html.markdown]: fix

**Pull request merged**